api/lib/source/polkadotV0-source.js

```diff
@@ -198,10 +198,9 @@ class polkadotAPI {
   async getBalancesV2FromAddress(address, fiatCurrency) {
     const api = await this.getAPI()
     const account = await api.query.system.account(address)
-    const totalBalance = account.data.free
-    const freeBalance = BigNumber(totalBalance.toString()).minus(
-      account.data.miscFrozen.toString()
-    )
+    const { free, feeFrozen } = account.data.toJSON()
+    const totalBalance = BigNumber(free)
+    const freeBalance = BigNumber(free).minus(feeFrozen)
```

Comment:
Does this fix our current UX problem with balances not changing after a delegation?

Reply:
I think though

Reply:
what is `feeFrozen`? it is not obvious from the name.

Reply:
Not obvious toe either. It's from the Polkadot API. It seems to hold the locked values. Need to monitor further.

Reply:
It is not working on my side... I just get reduced fees, like before (trying with Westend)

Reply:
Yes, I can confirm it is not working... not sure yet what `feeFrozen` stores but it is not the whole balance you aim to stake

Reply:
yes, it works! :tada: sorry for the confusion :laughing: